save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/TestSuitesSourceController.kt

```diff
@@ -0,0 +1,262 @@
+package com.saveourtool.save.backend.controllers
+
+import com.saveourtool.save.backend.ByteBufferFluxResponse
+import com.saveourtool.save.backend.service.*
+import com.saveourtool.save.backend.storage.TestSuitesSourceSnapshotStorage
+import com.saveourtool.save.entities.TestSuite
+import com.saveourtool.save.entities.TestSuitesSource
+import com.saveourtool.save.testsuite.*
+import com.saveourtool.save.utils.*
+import com.saveourtool.save.v1
+import org.slf4j.Logger
+import org.springframework.http.HttpStatus
+import org.springframework.http.MediaType
+import org.springframework.http.ResponseEntity
+import org.springframework.http.codec.multipart.Part
+import org.springframework.web.bind.annotation.*
+import reactor.core.publisher.Mono
+import reactor.kotlin.core.publisher.toFlux
+import reactor.kotlin.core.publisher.toMono
+
+typealias TestSuiteList = List<TestSuite>
+
+/**
+ * Controller for [TestSuitesSource]
+ */
+@RestController
+@RequestMapping("/internal/test-suites-sources", "/api/$v1/test-suites-sources")
+class TestSuitesSourceController(
+    private val testSuitesSourceService: TestSuitesSourceService,
+    private val testSuitesSourceSnapshotStorage: TestSuitesSourceSnapshotStorage,
+    private val testSuitesService: TestSuitesService,
+    private val organizationService: OrganizationService,
+    private val gitService: GitService,
+    private val executionService: ExecutionService,
+) {
+    /**
+     * @param organizationName
+     * @return list of [TestSuitesSourceDto] found by provided values or empty response
+     */
+    @GetMapping("/{organizationName}/list")
+    fun list(
+        @PathVariable organizationName: String,
+    ): Mono<TestSuitesSourceDtoList> = Mono.just(organizationName)
+        .flatMap {
+            organizationService.findByName(it).toMono()
+        }
+        .switchIfEmptyToResponseException(HttpStatus.CONFLICT) {
```

Comment:
Why `conflict`?

Reply:
`NOT_FOUND` can be indeed confused with the case when test suite source by requested parameters is not found. Ok, let's keep `CONFLICT` for missing organizations